.github/workflows/assign_reviewer.yml

```diff
@@ -2,15 +2,20 @@ name: Assign Reviewer
 
 on:
   pull_request:
-    types: ["labeled"]
+    types:
+      - opened
+      - reopened
+      - edited
+      - review_request_removed
+      - labeled
 
 permissions:
   pull-requests: write
 
 jobs:
   assign_reviewer:
     runs-on: ubuntu-latest
-    if: github.event.label.name == 'dependencies' && github.event.pull_request.user.login == 'app/renovate'
+    if: contains(github.event.pull_request.labels.*.name, 'dependencies') && github.event.pull_request.requested_reviewers == null && (github.event.pull_request.user.login == 'renovate[bot]')
```

Comment:
How does `github.event.pull_request.requested_reviewers == null` interact with auto-assigned code owners? The idea was that the workflow assigns a reviewer in addition to the code owners.

Reply:
I have not actually tested this, but from what I can gather `requested_reviewers` is a list of assigned reviewers, which is not empty if there are auto assigned reviewers.